### DISCUSSION TITLE
CodeActionsOnSave Support?

### DISCUSSION THREAD

**Discussion by JustinGrote** [prettier/prettier-vscode#2798] [Q&A]
Does Prettier support the newer `editor.CodeActionsOnSave` setting with the array syntax instead of the default formatter? Would using the `prettier.forceFormatDocument` be the command I want to use in this case?

**Comment by Airkro** [comment]
Using [Format Code Action](https://marketplace.visualstudio.com/items?itemName=rohit-gohri.format-code-action) or [Code Action Register](https://marketplace.visualstudio.com/items?itemName=FireWorks.code-action-register)
